ui.content/pom.xml

```diff
@@ -57,6 +57,18 @@
                             <destinationFile>${basedir}/src/main/content/jcr_root/conf/venia/settings/cloudconfigs/commerce/.content.xml</destinationFile>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>copy-cq-var-content</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Copies the right filter.xml file depending on the profile -->
+                            <sourceFile>${basedir}/src/main/content/filter-templates/filter-${project.classifier}.xml</sourceFile>
+                            <destinationFile>${basedir}/src/main/content/META-INF/vault/filter.xml</destinationFile>
```

Comment:
Shouldn't you remove the `filter.xml` file from git (remove and then add to `.gitignore`)? Otherwise each time we build locally for `classic`, this file might have changes and someone will end up pushing the change. That's why I removed the "changing" `content.xml` file, see the bottom of the `.gitignore` file.

Reply:
I removed the `filter.xml` from git, but didn't add it to .gitignore. I'll do this now.